### ISSUE TITLE
RuntimeWarning when using convert_magnet

### ISSUE THREAD

**Issue by andocromn** [Flexget/Flexget#1978]
### Actual behaviour:
Runtime warning is produced when plugin convert_magnet is instantiated 

### Steps to reproduce:
enable convert_magnet plugin

#### Config:
```
tasks:
  stage:
    convert_magnet: yes
    entry_list: listA
    accept_all: yes
    download: yes
```

#### Log:
```
2017-10-03 14:57 VERBOSE  manager                       Creating new database /home/user/test/db-config.sqlite - DO NOT INTERUPT ...
2017-10-03 14:57 VERBOSE  task_queue                    There are 1 tasks to execute. Shutdown will commence when they have completed.
/usr/local/lib/python2.7/dist-packages/flexget/plugins/modify/convert_magnet.py:70: RuntimeWarning: to-Python converter for boost::shared_ptr<libtorrent::alert> already registered; second conversion method ignored.
  import libtorrent  # noqa
2017-10-03 14:57 WARNING  details       stage          Task didn't produce any entries. This is likely due to a mis-configured or non-functional input.
2017-10-03 14:57 VERBOSE  details       stage          Summary - Accepted: 0 (Rejected: 0 Undecided: 0 Failed: 0)

```

### Additional information:

- FlexGet version: 2.10.95
- Python version: 2.7.13
- Installation method: pip
- Using daemon (yes/no): no
- OS and version: Debian Stretch 9.1
- Link to crash log: (Does Not Crash)

### Notes 
plugins/modify/convert_magnet.py contains 'import libtorrent' on line 34 and line 70 both

**Comment by andocromn** [comment]
I have tested with removing line 34 and line 70 individually and the warning occurs regardless.  So my initial assessment of the duplicate import causing the issue was incorrect.  I did some additional research and testing, looks like the issue is actually coming from within libtorrent, so not a flexget issue.

**Comment by andocromn** [comment]
for reference: Resolved by upgrading libtorrent from 1.1.1.0 to 1.1.4.0

**Comment by cvium** [comment]
Maybe this should be posted on the wiki
